.gitignore

```
@@ -14,6 +14,10 @@
 # Misc junk folder
 junk/
 
+# Helm dependency build generated files
+charts/*/charts
+chart.lock
```

I think this might be `Chart.lock`? I don't have such a file, but this doc supports my theory https://helm.sh/docs/helm/helm_dependency_build/
```suggestion
Chart.lock
```